typescript/tsconfig.node18.json

```diff
@@ -4,9 +4,9 @@
   "display": "Vercel Node 18",
   "docs": "https://github.com/vercel/style-guide",
   "compilerOptions": {
-    "lib": ["es2023"],
+    "lib": ["esnext"],
```

Comment:
I think lib is fine as is actually. seems to be an allowed value based on tsconfig json schema